streams/uy.m3u

```diff
@@ -1,8 +1,6 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="" status="error",CANAL 10
-http://edge2-ccast-sl.cvattv.com.ar/live/c5eds/Canal10_URU/verimatrix_rotating/Canal10_URU-video=1480000.m3u8
-#EXTINF:-1 tvg-id="" status="error",Canal 10 (720p)
-https://edge3-hr.cvattv.com.ar/live/c5eds/Canal10_URU/verimatrix_rotating_FTA/Canal10_URU.m3u8
+#EXTINF:-1 tvg-id="",Canal 10 (720p)
+https://edge3-hr.cvattv.com.ar/live/c4eds/Canal10_URU/SA_SAGEMCOM/Canal10_URU.m3u8
```

Comment:
I get this error in VLC:
<img width="610" alt="image" src="https://user-images.githubusercontent.com/7253922/158035036-ef9d7f3b-38fa-4a09-804f-b6cf35ed2f11.png">

Reply:
Aren't you in Russia it could be because CDN providers are actively trying to cut Russia out of internet access or speediness.
![image](https://user-images.githubusercontent.com/43685307/158038866-33210730-5d82-4f1b-820f-29cf58e7f369.png)

Reply:
> I get this error in VLC: <img alt="image" width="610" src="https://user-images.githubusercontent.com/7253922/158035036-ef9d7f3b-38fa-4a09-804f-b6cf35ed2f11.png">

Try with a VPN or proxy, recent Internet bans in Russia might have impacted certain streaming servers incl. this one.

Reply:
You're right, the problem was the geo-blocking.